### ISSUE TITLE
Please provide the ISSIM code or training log, or I will ask the TMM editorial department to review your article

### ISSUE THREAD

**Issue by dddddasda** [dk-liang/FIDTM#35]
I am unable to reproduce your results. Even with the ISSIM I produced, the results are far from yours.  Besides, I can't assure if the model pth you provided was trained on the test set. Please provide more evidence (training log or complete code).

**Comment by dk-liang** [comment]
Thanks!
Actually, we are preparing a general PyTorch-based codebase for crowd analysis named **OpenCrowd** (including mainstream methods like MCNN, DM-Count, IIM, TopoCount, CSRNET, BL, P2PNet, CAN, SANet, our work, and so on). The required code will be merged into this toolbox. We would like to release **OpenCrowd** when we think it is perfect (the current version is very rough). However, **OpenCrowd** only includes two people, so we do not have too much time to complete it. If you are free, we hope you can help us to finish **OpenCrowd**.

    
<img width="1544" alt="image" src="https://user-images.githubusercontent.com/23117298/221359959-0440ea27-5951-401c-8595-0e24559c2849.png">

<img width="1927" alt="image" src="https://user-images.githubusercontent.com/23117298/221360002-079bf8ba-11a1-46b0-b8c8-b97ea0d820f2.png">
<img width="1897" alt="image" src="https://user-images.githubusercontent.com/23117298/221360032-10e07474-dede-4f90-8edf-0065802ffbff.png">
<img width="1963" alt="image" src="https://user-images.githubusercontent.com/23117298/221360050-3a23ae7c-ef37-46c5-a5ec-8ddb91885027.png">

    

    
In addition, some people have successfully reproduced the results and even improved the FIDT:
(1) In the paper "Crowd Counting and Localization BeyondDensity Map", although the authors adopt VGG16 as the backbone, the performance of using FIDT with I-SSIM even surpasses our used HRNet in Counting on JHU-Crowd++. In addition, the localization performance also improved.

(2) In the paper "Counting in the 2020s: Binned Representations and inclusive Performance Measures for Deep CrowdCounting Approaches", the authors conduct extensive experiments and find that our method can surpass recent SOTA (P2PNet, ICCV 2021) in some settings. 

(3) In the paper "RDC‑SAL: Refne distance compensating with quantum scale‑awarelearning for crowd counting and localization", without SSIM, the authors achieve 62 MAE in ShanghaiA based on VGG-16, similar to our baseline (HRNet).

(4) In the paper "Dual reconstructive autoencoder for crowd localization and estimation in density and FIDT maps", the authors implement FIDT and I-SSIM in a single-column network.  

(5) In the paper "Cell Counting with Inverse Distance Kernel and Self-supervised Learning，" the authors adopt the FIDT in cell counting, and it works well.

(6) As a CVPR reviewer, I have reviewed a CVPR2023 submission. The authors also reproduce the FIDT with I-SSIM results (MAE slightly lower than our paper and localization better than our paper). Due to the CVPR policy, I could not tell you which paper it is.

(7) NWPU is the largest dataset to effectively reflect the performance of different methods. We want to emphasize that NWPU is a fair online benchmark, and **we can not obtain the label of the test set**. 

This is the three years ago projects. I do not save the training log since this was my first individual project when I was new to Pytorch. If you want the old version code, please refer [here](https://pan.baidu.com/s/1aPopkWBY83i_kHjubx590w?pwd=g3yc) (code:g3yc), which provides a simple early implementation (I have added some comments, but I do not have too much time to organize and check it). Note that although the old version contains the I-SSIM, you may need to change some hyper-parameters to achieve ideal performance. **I use Microsoft [NNI](https://github.com/microsoft/nni) to help choose the best hyper-parameters.**
